### PR TITLE
finish

### PR DESCRIPTION
**Pull request by devtanole**

### CODE REVIEW

File: typescript-conditionals/main.ts [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  } else {
+    return false;
+  }
+}
+
+function isOldEnoughToDrinkAndDrive(person: {
+  name: string;
+  age: number;
+}): any {
+  if (person.age > 0) {
```
Comment (hesam) [inline typescript-conditionals/main.ts:56]: Condition is incorrect. They want you to use AND operator to combine 2 conditions (drink and drive)

File: typescript-conditionals/main.ts [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+function startsWithJ(string: string): any {
+  if (string[0] === 'J') {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isOldEnoughToDrink(person: { name: string; age: number }): any {
```
Comment (hesam) [inline typescript-conditionals/main.ts:36]: Define interface Person beforehand, so that you don't have to do the thing inside function declaration

**Review by hesam — CHANGES_REQUESTED** [review]
Please see 2 comments inside code

File: typescript-conditionals/main.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+}
+function isOldEnoughToDrive(person) {
+  if (person.age >= 16) {
+    return true;
+  } else {
+    return false;
+  }
+}
+function isOldEnoughToDrinkAndDrive(person) {
+  if (person.age && person.name) {
```
Comment (hesam) [inline typescript-conditionals/main.js:48]: They just want you to combine the 2 conditions (person.age >= 21 and person.age >= 16) using and AND operator: &&

**Review by hesam — CHANGES_REQUESTED** [review]
Please see comment in code

**Review by hesam — APPROVED** [review]

File: typescript-conditionals/main.js (hunk 3 above)
Comment (devtanole) [reply typescript-conditionals/main.js:48]: Ah, I assumed it should've been set to false no matter what so I put it for if the property has any value in place.

**Pull request merged**